Index: spec/tumblr_spec.rb
===================================================================
--- spec/tumblr_spec.rb	(revision 75)
+++ spec/tumblr_spec.rb	(working copy)
@@ -5,7 +5,7 @@
 describe "Place your specs here" do
   
   it "find this spec in spec directory" do
-    violated "Be sure to write your specs"
+    violated "Be sure to write your specs : please test responsibly"
   end
   
-end
\ No newline at end of file
+end
Index: config/hoe.rb
===================================================================
--- config/hoe.rb	(revision 75)
+++ config/hoe.rb	(working copy)
@@ -47,7 +47,8 @@
 
 # Generate all the Rake tasks
 # Run 'rake -T' to see list of generated tasks (from gem root directory)
-hoe = Hoe.new(GEM_NAME, VERS) do |p|
+hoe = Hoe.spec(GEM_NAME) do |p|
+  p.version = VERS
   p.author = AUTHOR 
   p.description = DESCRIPTION
   p.email = EMAIL
Index: README.txt
===================================================================
--- README.txt	(revision 75)
+++ README.txt	(working copy)
@@ -1,6 +1,45 @@
-README for ruby-tumblr
-======================
+= ruby-tumblr
+
+ http://ruby-tumblr.rubyforge.org/
+
+== DESCRIPTION:
+
 ruby-tumblr is a library for tumblr API.
 
-Copyright (c) 2007 Keita Yamaguchi
+== FEATURES/PROBLEMS:
 
+Unknown
+
+== SYNOPSIS:
+
+<3 ruby + <3 tumblr
+
+== REQUIREMENTS:
+
+To run Rake:
+
+hoe
+newgem
+tzinfo
+pit 
+mechanize (libxml2-dev and libxslt-dev required)
+rspec
+
+== INSTALL:
+
+sudo gem install hoe newgem tzinfo pit rspec mechanize
+
+
+% pit set tumblr.com
+
+---
+site:
+username:
+password:
+
+== LICENSE:
+Copyright (c) 2007-2008 Keita Yamaguchi
+
+ruby-tumblr is released under the Ruby license.
+
+
